projects/isy-angular-widgets/assets/theme/_isyfact-variables.scss

```diff
@@ -60,13 +60,14 @@ $accordionHeaderPadding: 0.875rem !default;
 //accordion
 $accordionSpacing: 0px !default;
 
-// Variables in theme file _variables.scss adjusted to meet color requirements of the Isyfact design for the input field
+// Variables in theme file _variables.scss adjusted to meet color requirements of the IsyFact design
 
-////input
+//font-weight
+$isyfact-bold: bold !default;
```

Comment:
```suggestion
$isyfactInputLabelFontWeight: bold !default;

Reply:
Benennung in $isyfactFontWeight, da die Variable auch für andere font-weights genutzt werden kann